src/modules/graphics/opengl.c

```diff
@@ -1484,15 +1480,15 @@ void lovrTextureAllocate(Texture* texture, uint32_t width, uint32_t height, uint
 
   GLenum glFormat = convertTextureFormat(format);
   GLenum internalFormat = convertTextureFormatInternal(format, texture->srgb);
-#ifndef LOVR_WEBGL
+#ifdef LOVR_GL
```

Comment:
Just curious, why this change?

Reply:
`glTexStorage` is core in GLES3 and WebGL, but not in desktop GL.  We only need to see if texStorage exists on desktop GL, this ifdef must have been written when only GL/WebGL were supported.

Reply:
OK. So _GL is explicitly distinct from GLES